config/application.rb

```diff
@@ -14,17 +14,6 @@
 
 module Upaya
   class Application < Rails::Application
-    config.generators do |g|
-      g.test_framework :rspec,
-                       fixtures: true,
-                       view_specs: false,
-                       helper_specs: false,
-                       routing_specs: false,
-                       controller_specs: false,
-                       request_specs: false
-      g.fixture_replacement :factory_girl, dir: 'spec/factories'
-    end
```

Comment:
I think its fine yes, since they would have merged fine.  If you had rebased I think it would have removed those changes.   In general, if you are going to squash you might as well rebase too since you are already re-writing history.